### PR TITLE
#14: Add an option to configure country translation

### PR DESCRIPTION
**Pull request by Sylry**

### CODE REVIEW

File: Form/Type/AddressEmbeddableType.php [hunk 1]
```diff
@@ -74,11 +77,13 @@ public function configureOptions(OptionsResolver $resolver)
             'allowed_countries' => [], // After updating to Symfony 5.1, make this a filter list based on country code.
             'preferred_countries' => [],
             'default_country' => null,
+            'choice_translation_domain' => null,
```
Comment (daften) [inline Form/Type/AddressEmbeddableType.php]: The default in ChoiceType is true, why make the default TRUE here? See https://symfony.com/doc/current/reference/forms/types/choice.html#choice-translation-domain. If you make the default true here, you don't need the isset higher up and can just add it.

File: Form/Type/AddressEmbeddableType.php [hunk 2]
```diff
@@ -53,6 +53,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             $countryCodeOptions['choice_loader'] = null;
         }
         $countryCodeOptions['preferred_choices'] = $options['preferred_countries'];
+        if (isset($options['choice_translation_domain'])) {
```
Comment (daften) [inline Form/Type/AddressEmbeddableType.php]: See below, the isset can be avoided I think.

**Review by daften — CHANGES_REQUESTED** [review]

**Comment by Sylry** [comment]
Thanks for the feedback, i'll look at it before Wednesday (hope i)

**Comment by daften** [comment]
I can't ever guarantee looking at incoming PR's myself, so no worries, no pressure ;)

File: Form/Type/AddressEmbeddableType.php (hunk 2 above)
Comment (Sylry) [reply Form/Type/AddressEmbeddableType.php:56]: Well, in the doc, the default value is TRUE, but in the code, it is FALSE, isn't it? https://github.com/symfony/symfony/blob/6.1/src/Symfony/Component/Form/Extension/Core/Type/CountryType.php#L43

That's why i did that way

File: Form/Type/AddressEmbeddableType.php (hunk 1 above)
Comment (Sylry) [reply Form/Type/AddressEmbeddableType.php]: Well, in the doc, the default value is TRUE, but in the code, it is FALSE, isn't it? https://github.com/symfony/symfony/blob/6.1/src/Symfony/Component/Form/Extension/Core/Type/CountryType.php#L43

That's why i did that way

Comment (daften) [reply Form/Type/AddressEmbeddableType.php]: Yes, I was looking at the wrong base class, you're absolutely right!

Comment (daften) [reply Form/Type/AddressEmbeddableType.php]: Well, the default should still be the same afterwards though, meaning the default here should be false then. If you do that you follow the standards from above and you can remove the isset (see other comment)

**Comment by Sylry** [comment]
Thanks for the comments, they are done :-)

**Comment by daften** [comment]
Looks good, thanks for the contribution @Sylry , Let me get a release out for this :)

**Comment by daften** [comment]
0.1.1 was released so you can enjoy your work more easily ;)

**Comment by Sylry** [comment]
great, thanks!

**Pull request merged**